### ISSUE TITLE
Not setting Mixpanel People profile property at all since build released Nov 21

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-android#3022]
https://mixpanel.com/project/2202789/view/143365/app/insights/#G1UFR9kudDRr

Could legitimately means we didn't trigger code to set any profile property, but it has always been set prior to Nov 21, better to check. In the graph, when `Android Devices = false`, it means it was not set.

Setting a property would be done with something like `getPeople().set(xxx)`

**Comment by JamesSmartCell** [comment]
@hboon this should be fixed in the 3.61 release. We did some mixpanel butchery to try to get the app past the censors but it turned out it was an issue on the Google side as it affected a lot of other people.

None of this 'butchery' was on the main branch, which we have released as of a few days ago.

**Comment by hboon** [comment]
Related message: https://discord.com/channels/@me/801704763162755072/1060783474514087937

**Comment by JamesSmartCell** [comment]
@hboon it seems like this one can be closed now?

**Comment by hboon** [comment]
@JamesSmartCell no, check out that Mixpanel link in the Discord link (ouch!). Those users are still being logged without setting `Android Devices`, which is done automatically once we set any user properties. Could it be that we haven't explicitly set any user properties yet?

**Comment by seabornlee** [comment]
It's a string and `false` means not set.

![image](https://user-images.githubusercontent.com/735708/215250013-11c42c09-b1eb-4680-b142-019165b97530.png)

We only set this value when analytics enabled, so I think these users with false value are using noAnalytics version. 
```
if (preferenceRepository.isAnalyticsEnabled())
        {
            firebaseAnalytics.setUserId(uuid);
            mixpanelAPI.identify(uuid);
            mixpanelAPI.getPeople().identify(uuid);
```

**Comment by hboon** [comment]
Then these users/sessions shouldn’t appear in MixPanel in the first place, right?

**Comment by seabornlee** [comment]
> Then these users/sessions shouldn’t appear in Mixpanel in the first place, right?

I think it can be a way to know how many users are using no analytics version.

**Comment by hboon** [comment]
Sorry, I was not clear, I meant, if there is this code that kicks off the `if` branch with `noAnalytics`:

```
if (preferenceRepository.isAnalyticsEnabled())
        {
            firebaseAnalytics.setUserId(uuid);
            mixpanelAPI.identify(uuid);
            mixpanelAPI.getPeople().identify(uuid);
```

shouldn't it also not log to Mixpanel at all with `noAnalytics`? i.e either:

A. With analytics: log to Mixpanel, events + profiles
B. No analytics: don't hit Mixpanel at all

**Comment by seabornlee** [comment]
I think so, I guess the original intention of no analytics version is for users who live in regions where they can't access Google services. But to conform F-Droid's policy, it requires no data collecting too.

**Comment by hboon** [comment]
@seabornlee do we know if this was fixed?

**Comment by hboon** [comment]
Released ~20230410. To check back
